### ISSUE TITLE
Deprecate createsuperuser management command

### ISSUE THREAD

**Issue by rtibbles** [learningequality/kolibri#1985]
## Summary

`createsuperuser` will create a user type that is no longer usable within Kolibri, either disable it, or modify it to do something useful. Similarly with `changepassword`.

**Comment by ivanistheone** [comment]
Note this issue is related to need for some sort of tool for setting up Kolibri in a fully automated manner.

Richard said:

The simplest way would be to POST to this endpoint: https://github.com/learningequality/kolibri/blob/release-v0.6.x/kolibri/core/device/api.py#L17
With a payload of this shape: https://github.com/learningequality/kolibri/blob/release-v0.6.x/kolibri/core/device/serializers.py#L47
URL specified here: https://github.com/learningequality/kolibri/blob/release-v0.6.x/kolibri/core/device/api_urls.py


Confirmed the above API works and the following shell script can be used for provisioning:

```
export KOLIBRI_DEVICEPROVISION_FACILTY_NAME="Demo Facility"
export KOLIBRI_DEVICEPROVISION_PRESETS="formal" # other options "nonformal" "informal"
export KOLIBRI_DEVICEPROVISION_SUPERUSER_USERNAME="devowner"
export KOLIBRI_DEVICEPROVISION_SUPERUSER_PASSWORD="admin123"
# ('id', 'username', 'full_name', 'password', 'facility', 'roles')
export KOLIBRI_LANG="en"
export KOLIBRI_PORT=8000

generate_post_data()
{
  cat <<EOF
{
  "facility": {
    "name": "$KOLIBRI_DEVICEPROVISION_FACILTY_NAME"
  },
  "preset": "$KOLIBRI_DEVICEPROVISION_PRESETS",
  "superuser": {
    "username": "$KOLIBRI_DEVICEPROVISION_SUPERUSER_USERNAME",
    "password": "$KOLIBRI_DEVICEPROVISION_SUPERUSER_PASSWORD"
  },
  "language_id": "$KOLIBRI_LANG"
}
EOF
}

curl -i \
  -H "Content-Type:application/json" \
  --data "$(generate_post_data)" \
  -X POST \
  "http://localhost:$KOLIBRI_PORT/api/deviceprovision/"
```

Long term we can think about adding a management command analogous to:
```
{{kalite}} manage setup --username={{ kalite_admin_user }} --password={{ kalite_admin_password }} --noinput
```
but with facility/preset options...

**Comment by indirectlylit** [comment]
note that simply disabling is not an option - we need a way for users to recover access to their data if they forget their password

**Comment by rtibbles** [comment]
Fixed in #2287 (createsuperuser and changepassword now work seamlessly with our models).
